src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionServiceBase.java

```diff
@@ -102,7 +102,7 @@ protected NoticeIssuedTemplateBody createPayload(PreSubmitCallbackResponse<SscsC
             writeFinalDecisionBuilder.dateOfDecision(null);
         }
 
-        writeFinalDecisionBuilder.appellantName(buildName(caseData, true));
+        writeFinalDecisionBuilder.appellantName(buildName(caseData, false));
```

Comment:
Isn't this just going to add the appointee name?

Reply:
> Isn't this just going to add the appointee name?

    protected String buildName(SscsCaseData caseData, boolean displayAppointeeName) {
        if (displayAppointeeName && "yes".equalsIgnoreCase(caseData.getAppeal().getAppellant().getIsAppointee())
            && null != caseData.getAppeal().getAppellant().getAppointee()) {
            return WordUtils.capitalizeFully(caseData.getAppeal().getAppellant().getAppointee().getName()
                    .getFullNameNoTitle(), ' ', '.');
        }
        return WordUtils.capitalizeFully(caseData.getAppeal().getAppellant().getName()
                .getFullNameNoTitle(), ' ', '.');
    }

true -> appointee name
false -> appellant name

Reply:
Don't know why I read that backwards, is there a test for this change?

Reply:
there is already test for **buildName** which covers both scenario.

Reply:
additional check in test added